packages/forms-web-app/src/dynamic-forms/journey-factory.js

```diff
@@ -1,3 +1,16 @@
+// This is one of the major moments where each journey being
+// a class rather than an instance of a class really jumps out.
+
+// This should not exist.
+
+// Make Journey isomorphic then you can just construct one where
+// they're needed.
+
+// A good first step would be moving the use of getJourney into
+// a single piece of middleware since it's only used in handlers.
+// That way it'll be easier to clean up the use of getJourney
+// once Journey itself has been refactored.
```

Comment:
This is almost done https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/3782